### ISSUE TITLE
Why not set the learning rate of sem_net？

### ISSUE THREAD

**Issue by BCL123456-BAL** [Clear-3d/torch-ngp-semantic#1]
optimizer = lambda model: torch.optim.Adam(model.get_params(opt.lr), betas=(0.9, 0.99), eps=1e-15)

  # optimizer utils in network_sem.py
    def get_params(self, lr):

        params = [
            {'params': self.encoder.parameters(), 'lr': lr},
            {'params': self.sigma_net.parameters(), 'lr': lr},
            {'params': self.encoder_dir.parameters(), 'lr': lr},
            {'params': self.color_net.parameters(), 'lr': lr}, 
            #So why not add a line code
            #{'params': self.sem_net.parameters(), 'lr': lr}
        ]
        if self.bg_radius > 0:
            params.append({'params': self.encoder_bg.parameters(), 'lr': lr})
            params.append({'params': self.bg_net.parameters(), 'lr': lr})
        
        return params

**Comment by Clear-3d** [comment]
Sorry for the delay in response. The settings you mentioned are directly based on the original torch-ngp configuration. The "sem_net" branch is my addition, but I found that it works well without setting it. Therefore, I didn't attempt to modify it. Perhaps making modifications could lead to even better results. Thank you for bringing it up.
